request.go

```diff
@@ -18,6 +18,9 @@ import (
 	"time"
 )
 
+// requestValidity represents when a signed request is meant to expire
+const requestValidity = 10 * time.Second
```

Comment:
Too short, it should handle some gap for desynchronized clock. What about 10 minutes?

Reply:
We can also have this as a parameter with a default value so that a configuration file for a tool can set this value to something different.

Reply:
if you put it yourself, _egoscale_ won't override it.